scipy/stats/tests/test_binned_statistic.py

```diff
@@ -19,6 +19,7 @@ def setup_class(cls):
         cls.v = np.random.random(100)
         cls.X = np.random.random((100, 3))
         cls.w = np.random.random(100)
+        cls.u = np.random.random(100) + 1e6# for # gh-10126
```

Comment:
extra `#` in here? and pep8 issue on the spacing I think

Reply:
Moved comment to unit test proper